common/jenkins-agents/graalvm/ocp-config/is.yml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+kind: Template
+metadata:
+  name: jenkins-agent-graalvm
+objects:
+- apiVersion: v1
+  kind: ImageStream
+  metadata:
+    name: jenkins-agent-graalvm
+    labels:
+      app: jenkins-agent-graalvm
+  spec:
+    dockerImageRepository: jenkins-agent-graalvm
```

Comment:
`dockerImageRepository` shouldn't be used, only when you want to reference a DockerHub repo.